lib/Backend/JITTimeWorkItem.cpp

```diff
@@ -38,13 +38,6 @@ JITTimeWorkItem::IsLoopBody() const
     return Type() == JsLoopBodyWorkItemType;
 }
 
-bool
-JITTimeWorkItem::IsJitInDebugMode() const
-{
-    // TODO (michhol): flags?
-    return Js::Configuration::Global.EnableJitInDebugMode()
-        && m_workItemData->isJitInDebugMode;
-}
```

Comment:
This should be kept just as only returning the value of  `m_workItemData->isJitInDebugMode;` without the flag check.